Submodule LibreTransmitter contains modified content
diff --git a/LibreTransmitter/Features.swift b/LibreTransmitter/Features.swift
index e5f56f1..ab41a9a 100644
--- a/LibreTransmitter/Features.swift
+++ b/LibreTransmitter/Features.swift
@@ -17,7 +17,7 @@ public final class Features {
     static public var glucoseSettingsRequireAuthentication = false
     static public var alarmSettingsViewRequiresAuthentication = false
     
-    static public var allowsEditingFactoryCalibrationData = false
+    static public var allowsEditingFactoryCalibrationData = true
     
     static var phoneNFCAvailable: Bool {
         return NFCNDEFReaderSession.readingAvailable
